modules/ROOT/partials/configuration/documents_file_types.adoc

```diff
@@ -0,0 +1,145 @@
+====
+endif::[]
+ifndef::includedSection[]
+[NOTE]
+====
+**Priority Behavior**
+
+When `+documents_file_types+` is set, it acts as the source of truth for supported file types throughout the editor. This means:
+
+* The Link plugin uses this custom list instead of its default supported file types
+
+If `+documents_file_types+` is not set:
+
+* The Link plugin uses its default list of supported file types
+====
+
+endif::[]
+
+*Type:* `+Array+`
+
+ifeval::["{includedSection}" == "uploadcarePlugin"]
+*Default value:* `+undefined+`. The Media Optimizer plugin automatically configures this option when both the `+link+` plugin and the Media Optimizer plugin are enabled. The automatically configured list includes documents, iWork files, audio files, and archive files.
+endif::[]
+ifndef::includedSection[]
+*Default value:* `+undefined+`
+endif::[]
+
+ifeval::["{includedSection}" == "uploadcarePlugin"]
+=== Example: Using `documents_file_types` with Media Optimizer
+
+This example shows how to customize the file types when using the Media Optimizer plugin with the xref:link.adoc[Link] plugin:
+
+[source,js]
+----
+tinymce.init({
+  selector: 'textarea',
+  plugins: 'uploadcare link',
+  toolbar: 'uploadcare link',
+  uploadcare_public_key: '<your-public-key>',
+  documents_file_types: [
+    { mimeType: 'application/msword', extensions: [ 'doc' ] },
+    { mimeType: 'application/vnd.ms-excel', extensions: [ 'xls' ] },
+    { mimeType: 'application/vnd.ms-powerpoint', extensions: [ 'ppt', 'pps' ] },
+    { mimeType: 'application/vnd.openxmlformats-officedocument.wordprocessingml.document', extensions: [ 'docx' ] },
+    { mimeType: 'application/vnd.openxmlformats-officedocument.spreadsheetml.sheet', extensions: [ 'xlsx' ] },
+    { mimeType: 'application/vnd.openxmlformats-officedocument.presentationml.presentation', extensions: [ 'pptx' ] },
+    { mimeType: 'application/pdf', extensions: [ 'pdf' ] },
+    { mimeType: 'application/rtf', extensions: [ 'rtf' ] },
+    { mimeType: 'text/plain', extensions: [ 'txt' ] },
+    { mimeType: 'application/x-iwork-keynote-sffkey', extensions: [ 'key' ] },
+    { mimeType: 'application/x-iwork-pages-sffpages', extensions: [ 'pages' ] },
+    { mimeType: 'application/x-iwork-numbers-sffnumbers', extensions: [ 'numbers' ] },
+    { mimeType: 'audio/wav', extensions: [ 'wav', 'wave' ] },
+    { mimeType: 'audio/mpeg', extensions: [ 'mp3' ] },
+    { mimeType: 'audio/ogg', extensions: [ 'ogg', 'oga', 'ogx', 'ogm', 'spx', 'opus' ] },
+    { mimeType: 'application/zip', extensions: [ 'zip' ] },
+    { mimeType: 'application/gzip', extensions: [ 'gz', 'gzip' ] },
+    { mimeType: 'application/x-tar', extensions: [ 'tar' ] },
+    { mimeType: 'application/x-gtar', extensions: [ 'tar.gz', 'tgz' ] },
+    { mimeType: 'application/x-7z-compressed', extensions: [ '7z' ] },
+    { mimeType: 'application/x-rar-compressed', extensions: [ 'rar' ] }
```

Comment:
I'm not sure if here you meant to put the `uploadcare` default, but I did some change to fix something, I wrote the new default in TINY-13370 (in comments), have a check, but maybe wait for the final approval to be sure that it will be the last version